server/bundler/loader.js

```diff
@@ -114,14 +114,10 @@ function splitTemplate( path, section ) {
 }
 
 function getPathRegex( pathString ) {
-	var regex;
-
 	if ( pathString === '/' ) {
 		return JSON.stringify( pathString );
 	}
-
-	regex = utils.pathToRegExp( pathString );
-
+	const regex = utils.pathToRegExp( pathString );
 	return '/' + regex.toString().slice( 1, -1 ) + '/';
```

Comment:
not that this was super clear before, but could we annotate what we're trying to accomplish here? maybe add a comment with a before/after string… I find that the slice is elusive and I couldn't feel confident working with this code because I don't know what it's supposed to actually do